---
layout: page
title: Research
description: ---
---

Synthetic biology is a rapidly growing field set to make profound impact in numerous industries including manufacturing, healthcare, agriculture, and sustainable energy, as well as our fundamental understanding of life itself. Progress has been hindered, however, by the difficulty in engineering biological systems reliably. One promising approach is to use cell-free expression systems. These are *in vitro* systems that mimic the cellular environment and can be integrated with microfluidic technologies to rapidly prototype and screen synthetic gene circuits, prior to their deployment *in vivo*. 

**PhD Opportunities**

We have three PhD projects available. Funding deadlines are **December 2019** for a start in autumn 2020. If you are interested in any of the projects below please send a CV, a letter explaining your interest in the project, and the names of two referees directly to [Dr Laohakunakorn](mailto:nadanai.laohakunakorn@ed.ac.uk). 

---

**1. Quantitative models for optimised cell-free gene expression** 

The quantitative nature of data from cell-free experiments, as well as the amount of control over perturbations and composition of the system allow access to internal parameters otherwise hidden in living cells. This enables precise mathematical modelling and inference of the system’s behaviour, and opens up possibilities for truly rational approaches to gene circuit design. 

The student will combine high-throughput microfluidics with model-building, parameter inference, and optimal experimental design strategies to develop a complete quantitative description of cell-free gene expression. In addition to developing deeper understanding of gene expression biophysics, the models will provide a quantitative platform for the optimisation of cell-free systems, with important implications for cell-free bioproduction of high-value chemicals.

The ideal candidate will be highly motivated, and possess a strong quantitative background in physics/engineering/computer science or related disciplines. Depending on the candidate’s interests, it will be possible to work on both computational and experimental aspects of the project. Experience in any of the following areas is highly desirable: numerical modelling, Bayesian inference, optimal experimental design, molecular biology, microfluidics.

This project will be jointly supervised with Dr Diego Oyarzún, who leads the [Biomolecular Control Group](http://homepages.inf.ed.ac.uk/doyarzun/).

---

**2. Mapping cell-free to cellular systems for improved rapid prototyping**

Cell-free systems have not yet fulfilled their potential as prototyping platforms, due to lack of data and understanding of the differences between the cell-free 'chassis' and a real host cell. This project aims to meet that need by developing automated characterisation of a large number of common parts and circuits in both cell-free and living cells, using the resources available at the [Edinburgh Genome Foundry](https://www.genomefoundry.org/). Identifying which parts behave consistently across the two platforms will allow cell-free systems to be more reliably used for rapid prototyping; and seeing where the differences are may point towards more fundamental gaps in our understanding of synthetic biology.

The ideal candidate will be a highly motivated biologist interested in quantitative approaches to gene circuit engineering, or a physicist/engineer interested in applying their skills to a wet-lab environment. Experience in any of the following areas is highly desirable: molecular biology, microbiology and cell culture, microfluidics, microscopy, programming, lab automation.  

This project will be jointly supervised with [Dr Filippo Menolascina](https://www.eng.ed.ac.uk/about/people/dr-filippo-menolascina) from the School of Engineering.

---

**3. Programmable transcriptional activation engineering in cell-free expression systems**

While cell free systems have already been used to prototype a large number of gene circuits using transcriptional repressors, building functional activators has remained elusive, despite their fundamental role in living cells. This project plans to deploy a number of different approaches from *in vivo* activator engineering and programmable gene regulation to build a repertoire of cell-free activators, forward-engineer a new class of gene circuits based on these parts, and understand the properties and limitations of these newly-engineered cell-free systems. 

The ideal candidate will be a highly motivated biologist interested in quantitative approaches to gene circuit engineering, or a physicist/engineer interested in applying their skills to a wet-lab environment. Experience in any of the following areas is highly desirable: molecular biology, cell culture, microfluidics, microscopy, programming. 

This project will be jointly supervised with [Dr Baojun Wang](http://wang.bio.ed.ac.uk/).


